File: nachos/src/test/java/com/hootsuite/nachos/CustomRobolectricRunner.java
package com.hootsuite.nachos;

import org.junit.runners.model.InitializationError;
import org.robolectric.RobolectricTestRunner;

/**
 * Custom Robolectric test runner for Nachos tests.
 * Updated for modern Robolectric version.
 */
public class CustomRobolectricRunner extends RobolectricTestRunner {

    public CustomRobolectricRunner(Class<?> testClass) throws InitializationError {
        super(testClass);
    }
}

File: nachos/src/test/java/com/hootsuite/nachos/CustomRobolectricRunner.kt
package com.hootsuite.nachos

import org.robolectric.RobolectricTestRunner

/**
 * Kotlin replacement for the custom Robolectric test runner used by Nachos unit tests.
 */
class CustomRobolectricRunner(testClass: Class<*>) : RobolectricTestRunner(testClass) 